src/App.tsx

```diff
@@ -53,12 +53,12 @@ const App = () => {
   const pendingTxHashes = useAppSelector((s) => selectAddressesPendingTransactions(s, addressHashes)).map(
     (tx) => tx.address.hash
   )
-  const [settings, network, addressesStatus, isPassphraseUsed, tokens] = useAppSelector((s) => [
+  const [settings, network, addressesStatus, isPassphraseUsed, assetsInfo] = useAppSelector((s) => [
     s.settings,
     s.network,
     s.addresses.status,
     s.activeWallet.isPassphraseUsed,
-    s.tokens
+    s.assetsInfo
```

Comment:
I decided to rename the slice because `asset` is a more accurate name than `token`, since ALPH is not a token, but a coin. Adding the ALPH info (decimals, name, symbol, etc) in the assets slice simplifies our code a lot (this way we don't have to add special cases to treat ALPH separately from tokens. They are both "assets" :)

Reply:
Yes, I thought about this as well as I was designing. That's perfect 💯